modules/fedimint-mint-client/src/lib.rs

```diff
@@ -861,7 +883,7 @@ impl MintClientModule {
         operation_id: OperationId,
     ) -> anyhow::Result<Vec<ClientInput<MintInput, MintClientStateMachines>>> {
         let notes_by_denomination: BTreeMap<Amount, Vec<SpendableNote>> = dbtx
-            .find_by_prefix_sorted_descending(&NoteKeyPrefix)
+            .find_by_prefix(&NoteKeyPrefix)
```

Comment:
Does the order matter at all here?

Reply:
Nope, but the natural order might be just easier for the db to do.